ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/TestSpecFactory.java

```diff
@@ -343,19 +343,30 @@ private static SpecConfigDeneb getDenebSpecConfig(final Eth2Network network) {
     return getDenebSpecConfig(network, UInt64.ZERO, UInt64.ZERO, UInt64.ZERO, UInt64.ZERO);
   }
 
-  private static SpecConfigDeneb getDenebSpecConfig(
+  public static SpecConfigDeneb getDenebSpecConfig(
       final Eth2Network network,
       final UInt64 altairForkEpoch,
       final UInt64 bellatrixForkEpoch,
       final UInt64 capellaForkEpoch,
       final UInt64 denebForkEpoch) {
+    return getDenebSpecConfig(
+        network, altairForkEpoch, bellatrixForkEpoch, capellaForkEpoch, denebForkEpoch, true);
+  }
+
+  public static SpecConfigDeneb getDenebSpecConfig(
+      final Eth2Network network,
+      final UInt64 altairForkEpoch,
+      final UInt64 bellatrixForkEpoch,
+      final UInt64 capellaForkEpoch,
+      final UInt64 denebForkEpoch,
+      final boolean kzgNoop) {
```

Comment:
seems very curious we've had to make this change? doesnt seem related to anything we're doing...